addons/website/tests/test_page_manager.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+@odoo.tests.common.tagged('post_install', '-at_install')
```

Comment:
Just to be sure, you have a python linter enabled and disabled the E302 rule (as per the officiel doc so it's fine even if I prefer to keep it myself)?

Reply:
Yes, the E302 rule is disabled in my python linter (as the E301 and the E501)